content/working_with/console/widgets/deploymentsView.md

```diff
@@ -0,0 +1,279 @@
+---
+layout: bt_wiki
+title: Deployments View
+category: Widgets
+draft: false
+---
+
+Presents information about deployments in a single view. Contains:
+
+- a table showing the deployments in the system (the left side) The deployments
+  can be further filtered.
+  See [Filtering deployments]({{< relref "#filtering-deployments" >}}).
+- a pane with the deployment details divided into tabs (the right side)
+- a map showing the deployments locations
+
+<!-- TODO: screenshot -->
+
+## Table
+
+The table shows deployments that match the current filter.
+See [Filtering deployments]({{< relref "#filtering-deployments" >}})
+for more details on filtering.
+
+The columns shown in the table can be shown and hidden using the widget's
+configuration.
+
+<!--  TODO: screenshot -->
+
+The columns can show:
+
+1. The deployment status represented as an icon
+
+   - No icon - _good_
+   - Yellow spinner - _in progress_
+   - Red exclamation mark - _requires attention_
+
+   See [the Environments and Services section]({{< relref "#environments-and-services" >}})
+   for more information on deployment statuses.
+
+2. ID - the deployment ID
+
+   You can see the exact ID by hovering over the _ID_ button. From the popup
+   you can also copy the ID directly to your clipboard.
+
+3. Name - the deployment display name
+4. Blueprint Name - the name of the blueprint that this deployment is using
+5. Environment Type - the value of the `csys-env-type` label for that
+   deployment
```

Comment:
Do we have reserved keys described somewhere? If so, then we can probably reference it here.

Reply:
I did a quick look at the docs and I did not find them mentioned anywhere :worried: As I grepped, this is the only place in the code that mentions `csys-env-type`

Reply:
@qooban, this is a great question. I can think of reasons for why having such a page and why not.
Pros:
1. Users will be able to use reserved labels if they wish, and we need to provide them information on how to do so.
2. Users are not allowed to use non-reserved `csys-` prefixed labels, so might as well provide them information on the ones that are reserved and **can** be used.  

Cons:
1. The documentation will have to be updated with each addition/deletion of a reserved label, and I don't know if that's something we can expect to be done. 
2. Some of the reserved labels are context-specific, e.g. `csys-wrcp-services`, so are not relevant for all users.

These are just my 2 cents on it, it would be up to @ofercloudify's decision.